lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -637,12 +647,13 @@ def cmd_transport(*args)
     }
 
     valid = true
+    transport_index = 0
     @@transport_opts.parse(args) do |opt, idx, val|
       case opt
       when '-c'
         opts[:cert] = val
-      when '-u'
```

Comment:
So this means we can't remove transports the old way? I'd prefer to keep both. If I've added a transport, it's handy to be able to remove it using the same syntax.

I think supporting both shouldn't be an issue. What do you think?